docs/how-to-enable-new-languages.md

```diff
@@ -418,4 +418,7 @@ Then commit and push your branch directly to the News repo.
 
 Finally, open a PR for review.
 
-Once both your PRs to the CDN and News repo have been approved, they can be merged. Deployment will be handled subsequently by the staff.
+Once both your PRs to the CDN and News repo have been approved, they can be merged. 
+
+> [!NOTE]
+> Deployment will be handled subsequently by the staff. Here is a sample PR: [freeCodeCamp/news#485](https://github.com/freeCodeCamp/news/pull/485) of how they do it and more details are available in the [staff-wiki](https://staff-wiki.freecodecamp.org/docs/flight-manuals/news-instances#jamstack---news--assets).
```

Comment:
I thought staff-wiki was access controlled by `@freecodecamp.org`?

Reply:
It is; you are welcome to test.

Reply:
What I meant is should it be linked to if majority cannot view it?

Reply:
<img src="https://user-images.githubusercontent.com/1884376/198351259-8d46e50e-1fcd-4f27-8a1c-73748f636d66.png" width="300">

Reply:
This is where it is linked to. I am open to ideas though, once you give the current guide a read.

Thanks.